lib/bigshift/redshift_table_schema.rb

```diff
@@ -8,7 +8,17 @@ def initialize(schema_name, table_name, redshift_connection)
 
     def columns
       @columns ||= begin
-        rows = @redshift_connection.exec_params(%|SELECT "column", "type", "notnull" FROM "pg_table_def" WHERE "schemaname" = $1 AND "tablename" = $2|, [@schema_name, @table_name])
+        query = p %{
+          SELECT "column", "type", "notnull"
+          FROM "pg_table_def" ptd, information_schema.columns isc
+          WHERE ptd.schemaname = isc.table_schema
+          AND ptd.tablename = isc.table_name
+          AND ptd.column = isc.column_name
+          AND "schemaname" = $1
+          AND "tablename" = $2
+          ORDER BY ordinal_position
+        }.gsub(/\s+/, " ").strip
```

Comment:
The space could be single quoted to show that no string interpolation is necessary.